packages/math-input/src/components/input/math-input.js

```diff
@@ -651,7 +630,7 @@ class MathInput extends React.Component<Props, State> {
         }
     };
 
-    handleTouchEnd: ($FlowFixMe) => void = (e) => {
+    handleTouchEnd: (SyntheticTouchEvent<HTMLDivElement>) => void = (e) => {
```

Comment:
The other touch handles use `HTMLSpanElement`.  Should these be the same?  If so, I wonder if we should declare a type alias at the top of the file that represents the element type.  Ideally I'd love to type this as the result of the `render()` call, but I don't think it's easy/possible to "extract" that type using Flow utility types. 

At a min let's just make sure we have the right DOM element type across all the event registrations. 

Given that we don't refer to the target in many of these, I wonder if we can just do `SyntheticTouchEvent<*>`.. 🤔

Reply:
There are two different inputs that are triggering touch events:
- The `View` is the container for the math-input and is a `div`
- The `CursorHandle` is another interactive element and is a `span`